app/controllers/api/v1/messages_controller.rb

```diff
@@ -0,0 +1,42 @@
+module Api
+  module V1
+    class MessagesController < Api::V1::ApiController
+      def index
+        @messages = policy_scope(conversation.messages).includes(:user)
+      end
+
+      def create
+        @conversation = conversation
```

Comment:
try to avoid more than one instance variable in controllers at least

Reply:
I modify the controller, but let me know if there is something more that i can improve here. Thanks!

Reply:
Y left another comment, my suggestion was to remove the @conversation only not @message that is required for the create view